terraform/modules/read-only-users/main.tf

```diff
@@ -0,0 +1,83 @@
+# You can define readonly usernames, and their keybase key if applicable, below, and this will automatically create:
+# - their account
+# - an attachment to the "readonly" group, which has the IAM policy of readonly and can assume readonly roles, and forced MFA
+# - if a keybase key is provided, it will also create their user login profile
+locals {
```

Comment:
How does the module react to the user list being empty?

Reply:
I expect it will fail, but I think this will be temporary code and removed after the health check (famous last words 😂 )